dali/test/python/test_utils.py

```diff
@@ -69,12 +77,15 @@ def save_image(image, file_name):
         elif min >= -128 and max <= 127:
             image = image + 128
     else:
-        image = (image - np.iinfo(image.dtype).min) * (255.0 / (np.iinfo(image.dtype).max - np.iinfo(image.dtype).min))
+        image = (image - np.iinfo(image.dtype).min) * (255.0 /
+                                                       (np.iinfo(image.dtype).max - np.iinfo(image.dtype).min))
```

Comment:
WTF? Some crazy autoformatter or what?
```suggestion
        image = (image - np.iinfo(image.dtype).min) *
                      (255.0 /  (np.iinfo(image.dtype).max - np.iinfo(image.dtype).min))
```

Reply:
In general, please don't reformat whole files.... Please?

Reply:
Looks like crazy autoformatter. Had to turn of Python ones as they didn't want to behave or format changes only. Will fix.